src/languages/en.ts

```diff
@@ -2435,22 +2435,23 @@ export default {
             syncNow: 'Sync now',
             disconnect: 'Disconnect',
             disconnectTitle: (integration?: ConnectionName): string => {
-                switch (integration) {
-                    case CONST.POLICY.CONNECTIONS.NAME.QBO:
-                        return 'Disconnect QuickBooks Online';
-                    case CONST.POLICY.CONNECTIONS.NAME.XERO:
-                        return 'Disconnect Xero';
-                    default: {
-                        return 'Disconnect integration';
-                    }
-                }
+                const integrationName = integration && CONST.POLICY.CONNECTIONS.NAME_MAP[integration] ? CONST.POLICY.CONNECTIONS.NAME_MAP[integration] : 'integration';
+                return `Disconnect ${integrationName}`;
```

Comment:
Can this be simplified down to: 
```suggestion
                return `Disconnect ${CONST.POLICY.CONNECTIONS.NAME_MAP[integration] ?? 'integration'}`;
```

Reply:
Get a TS error. Hence I had to use this way.

<img width="1136" alt="Screenshot 2024-06-28 at 2 20 50 AM" src="https://github.com/Expensify/App/assets/3069065/32f6aeba-343a-4f8a-a877-244025bb5204">

Reply:
Can we just be strict with the types for the function?
```JS
            disconnectPrompt: (currentIntegration: ConnectionName): string => `Are you sure you want to disconnect ${CONST.POLICY.CONNECTIONS.NAME_USER_FRIENDLY[currentIntegration] ?? 'integration'}?`},

```

Reply:
Good point, let me check if it has any other usage. I can update. Give me a few mins.

Reply:
I was able to update it for `connectTitle` but `disconnectTitle` needs to allow undefined.

`Argument of type 'PolicyConnectionName | undefined' is not assignable to parameter of type 'keyof Connections'.\n  Type 'undefined' is not assignable to type 'keyof Connections'.`